README.md

```diff
@@ -6,74 +6,51 @@ C#/WinRT provides Windows Runtime (WinRT) projection support for the C# language
 
 WinRT APIs are defined in `*.winmd` format, and C#/WinRT includes tooling that generates C# code that can be compiled into interop assemblies, similar to how [C++/WinRT](https://github.com/Microsoft/cppwinrt) generates headers for the C++ language projection. This means that neither the C# compiler nor the .NET Runtime require built-in knowledge of WinRT any longer.
```

Comment:
Maybe "includes tooling that generates C# code for consumption scenarios, or generates a `.winmd` for authoring scenarios. Generated C# source code can be compiled into interop assemblies, ..."

Reply:
added